lxd/loki/loki.go

```diff
@@ -287,21 +287,25 @@ func (c *Client) HandleEvent(event api.Event) {
 		}
 
 		entry.Line = fmt.Sprintf("%s%s", messagePrefix, lifecycleEvent.Action)
-	} else if event.Type == api.EventTypeLogging {
+	} else if event.Type == api.EventTypeLogging || event.Type == api.EventTypeOVN {
 		logEvent := api.EventLogging{}
 
 		err := json.Unmarshal(event.Metadata, &logEvent)
 		if err != nil {
 			return
 		}
 
-		// The errors can be ignored as the values are validated elsewhere.
-		l1, _ := logrus.ParseLevel(logEvent.Level)
-		l2, _ := logrus.ParseLevel(c.cfg.logLevel)
+		// Check the log level only when dealing with logging events.
+		// For OVN events, the log level can be changed in the OVN config.
```

Comment:
Do the OVN logs an a log level that can be parsed?

I wonder if we should use that and only send to loki if matching the associated configured LXD log level to avoid spamming loki if debug is turned on in OVN, for example.

Reply:
> Do the OVN logs an a log level that can be parsed?

They do. The [man page for ovs-appctl](https://man7.org/linux/man-pages/man8/ovs-appctl.8.html) lists `emer`, `err`, `warn`, `info`, and `debug`.

> I wonder if we should use that and only send to loki if matching the associated configured LXD log level to avoid spamming loki if debug is turned on in OVN, for example.

Fine with me, as the log levels are basically the same, except for `emer`.